### PR TITLE
Prettier setup 

### PR DESCRIPTION
**Pull request by jiji14**
**This PR follows #1068.** 
**To review all the discussions regarding the formatting, please refer to the previous PR.** 
**I created a new PR to maintain a clean and readable branch.**

<br/>

### Related Issue
For considerations regarding Prettier, please refer to the following GitHub issue: [Prettier Issue #1014](https://github.com/e-mission/e-mission-docs/issues/1014).

<br/>

### Configuration Options
Our team has carefully chosen these configuration options, which are based on discussions. 
To explore more details, please visit the [Prettier documentation on options](https://prettier.io/docs/en/options).

<br/>

### How to Run Prettier
You can format your code using Prettier by running the following command:
```
npx prettier --write {filename}
```

<br/>

### Auto-Saving

If you wish to enable auto-saving locally, please follow these steps:

1. Create a .vscode folder at the project's root directory.
2. Inside the .vscode folder, create a settings.json file and include the following code:
```
{
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true
}
```
3. Additionally, install the `Prettier - Code formatter` VSCode extension.

<br/>

#### The next PR will set up GitHub Actions for Prettier.

### CODE REVIEW

**Review by JGreenlee — APPROVED** [review]
Great work! LGTM

**Comment by shankari** [comment]
@JGreenlee @jiji14 this is cool, but we need to figure out the timing.

Once I push the onboarding branch to production, I will attempt to merge it into the current master, at which point, we will have hundreds of merge conflicts.

I am tempted to wait until I have merged onboarding, and have you re-run prettier.
Thoughts?

**Comment by shankari** [comment]
@jiji14 merge is now complete! However, the `service_rewrite_2023` branch has significantly diverged from master.
So I would suggest the following flow:
- everybody pulls from `service_rewrite_2023` to their open/pending branches and resolve conflicts
- you run prettier on `service_rewrite_2023`
- people again pull from `service_rewrite_2023` to their open/pending branches and resolve conflicts

When we next pull from `service_rewrite_2023` to `master`, the prettier changes will come along by default

**Pull request closed without merging**
